reference-artifacts/SAMPLE_CONFIGS/config.example.json

```diff
@@ -93,7 +93,21 @@
         "High": ["myemail+notifyT-high@example.com"],
         "Medium": ["myemail+notifyT-medium@example.com"],
         "Low": ["myemail+notifyT-low@example.com"]
-      }
+      },
+      "dynamic-s3-log-partitioning": [
+        {
+          "logGroupPattern": "MAD",
+          "s3Prefix": "managed-ad"
+        },
+        {
+          "logGroupPattern": "rql",
+          "s3Prefix": "rql"
+        },
+        {
+          "logGroupPattern": "SecurityHub",
+          "s3Prefix": "security-hub"
+        }        
```

Comment:
Can we add these 3 additional log types:
,
        {
          "logGroupPattern": "Nfw",
          "s3Prefix": "nfw"
        },
        {
          "logGroupPattern": "rsyslog",
          "s3Prefix": "rsyslog"
        },
        {
          "logGroupPattern": "SSM",
          "s3Prefix": "ssm"
        }